### PR TITLE
Add universal CGO build image.

### PR DESCRIPTION
**Pull request by SuperQ**
Make a single Docker image for all builds.
* Merge crossbuild packages into one Dockerfile.
* Update builder script to support all CGO crossbuilds.
* Add tags to point to main image for compatibility.
* Use crossbuild for 386 CGO builds.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
I made this switch from stable to buster explicitly to avoid unexpected change when bullseye is released.

**Comment by SuperQ** [comment]
Ok, this is working and ready to go.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,21 +1,28 @@
 ---
-version: 2
+version: 2.1
+
+executors:
+  golang:
+    docker:
+    - image: cimg/go:1.16
```
Comment (roidelapluie) [inline .circleci/config.yml:7]: Does it mean we do not pick the golang minor release anymore ?

Comment (SuperQ) [reply .circleci/config.yml:7]: This is image used to run this build. Most of which is just calling docker commands. The only reason we need a golang image here is for the bump_version workflow.

**Review by roidelapluie — APPROVED** [review]
LGTM, thanks!

**Pull request merged**